src/kirby/components/segmented-control/segmented-control.component.html

```diff
@@ -0,0 +1,12 @@
+<ion-segment mode="ios" (ionChange)="onSegmentClick($event)">
+  <ng-container *ngFor="let item of segmentItems">
+    <div class="segment-btn-wrapper">
+      <ng-container *ngIf="item.badge">
+        <kirby-badge attr.colortype="{{item.badge.type}}" >{{item.badge.content}}</kirby-badge>
+      </ng-container>
+      <ion-segment-button value="{{item.id}}" class="kirby-text-small" checked="{{item.checked}}">
```

Comment:
Consider using property bindings, eg. `[value]="item.id"` (and `checked`-attribute) instead of text-interpolation, since no text is actually being interpolated.